corehq/apps/reports/standard/inspect.py

```diff
@@ -102,20 +108,20 @@ def es_query(self):
         query = (form_es.FormES()
                  .domain(self.domain)
                  .filter(time_filter(gte=self.datespan.startdate,
-                                     lt=self.datespan.enddate_adjusted))
-                 .filter(self._get_users_filter(mobile_user_and_group_slugs)))
+                                     lt=self.datespan.enddate_adjusted)))
+
+        users_filter = self._get_users_filter(mobile_user_and_group_slugs)
+        if users_filter:
+            query = query.filter(users_filter)
 
         # filter results by app and xmlns if applicable
         if FormsByApplicationFilter.has_selections(self.request):
             form_values = list(self.all_relevant_forms.values())
             if form_values:
                 query = query.OR(*[self._form_filter(f) for f in form_values])
+        else:
```

Comment:
aside: `FormsByApplicationFilter.has_selections` counts "Show Advanced Options" as a selection, which is a little unintuitive in my opinion.